src/test/scala/nl/knaw/dans/easy/fedoratobag/AppSpec.scala

```diff
@@ -165,10 +175,42 @@ class AppSpec extends TestSupportFixture with FileFoXmlSupport with BagIndexSupp
     )(CsvRecord.csvFormat.print(sw)) shouldBe Success("no fedora/IO errors")
     sw.toString should fullyMatch regex
       """easyDatasetId,uuid1,uuid2,doi,depositor,transformationType,comment
-        |easy-dataset:17,.+,,,,-,FAILED: java.lang.Exception: checksum error .* easy-file:35 .*/data/original/something.txt
+        |easy-dataset:17,.+,,,,-,FAILED: java.lang.Exception: Different checksums in fedora Some(.*) and exported bag Some(.*) for .*/data/original/something.txt
         |""".stripMargin
   }
 
+  it should "report a duplicate file" in {
+    val app: AppWithMockedServices = new AppWithMockedServices() {
+      expectAUser()
+      Map(
+        "easy-dataset:13" -> XML.loadFile((sampleFoXML / "streaming.xml").toJava),
+        "easy-discipline:6" -> audienceFoXML("easy-discipline:6", "D35400"),
+        "easy-file:1" -> fileFoXml(id = 1, location = "original/a", name = "x.txt", digest = digests("acabadabra")),
+        "easy-file:2" -> fileFoXml(id = 2, location = "a", name = "x.txt", digest = digests("acabadabra")),
```

Comment:
is the fact they have the same checksum relevant? Or would it produce the same error if the checksum was different?
This would be useful info for the datamanager how will need to fix this input

Reply:
It used to be relevant half way this pull request, three days ago.